<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>659</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Rocket Designer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Main Background Color</string>
    </property>
    <property name="title">
     <string>Base Color</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>r</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>g</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>b</string>
     </property>
    </widget>
    <widget class="QWidget" name="baseColourPanel" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="baseColourPickerButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>81</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Color Picker</string>
     </property>
    </widget>
    <widget class="QLabel" name="baseColourLabel_r">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="baseColourLabel_a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>58</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="baseColourLabel_b">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>58</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="baseColourLabel_g">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>58</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>a</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>450</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Color of the Active Page Indicator</string>
    </property>
    <property name="title">
     <string>Secondary Color</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>r</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>g</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>b</string>
     </property>
    </widget>
    <widget class="QWidget" name="secondaryColourPanel" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="secondaryColourPickerButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>81</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Color Picker</string>
     </property>
    </widget>
    <widget class="QLabel" name="secondaryColourLabel_r">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="secondaryColourLabel_g">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>58</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="secondaryColourLabel_b">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>58</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="secondaryColourLabel_a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>58</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>a</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>471</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Grid Properties</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Columns</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="columnSpin">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>72</y>
       <width>52</width>
       <height>32</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="rowSpin">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>32</y>
       <width>52</width>
       <height>32</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Rows</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="horizontalInversionCheckBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Inverted Horizontal Scrolling</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="verticalInversionCheckBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Inverted Vertical Scrolling</string>
     </property>
    </widget>
    <widget class="QComboBox" name="gridOrientationCombo">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>112</y>
       <width>271</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Horizontal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vertical</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Grid Orientation</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="systemWallpaperCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Use System Wallpaper</string>
     </property>
    </widget>
    <widget class="QPushButton" name="wallpaperSelectorButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>171</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Custom Wallpaper</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="systemWallpaperScreenSpin">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>186</y>
       <width>52</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="systemWallpaperScreenLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>186</y>
       <width>151</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Use Wallpaper of Screen</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentWallpaperLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>217</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Wallpaper:</string>
     </property>
    </widget>
    <widget class="QLabel" name="wallpaperLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>237</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="systemWallpaperScreenLabel_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>151</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Sets the blur effect. 0 means the background will not be blurred.</string>
     </property>
     <property name="text">
      <string>Background Blur Radius</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="systemWallpaperBlurSpin">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>270</y>
       <width>52</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="enableBoxesCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>154</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>If enabled items are shown in boxes; otherwise the whole background will be blurred with the base color</string>
     </property>
     <property name="text">
      <string>Enable Boxes</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>451</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Font Size</string>
     </property>
     <widget class="QSpinBox" name="font2Spin">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>32</y>
        <width>52</width>
        <height>32</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Font Size of the &quot;No results found&quot; Message</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="font1Spin">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>32</y>
        <width>52</width>
        <height>32</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Font Size of the Icons and of the Search Bar</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Font Size of the &quot;No results found&quot; Message</string>
      </property>
      <property name="text">
       <string>Secondary</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Font Size of the Icons and of the Search Bar</string>
      </property>
      <property name="text">
       <string>Primary</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>471</width>
      <height>34</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
    </property>
    <property name="centerButtons">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="rocketMetaCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Rocket if the Meta Key is Pressed</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>450</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Color of the Selected Item and of the Page Indicators, if Boxes are Disabled</string>
    </property>
    <property name="title">
     <string>Selection Color</string>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>r</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>g</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>b</string>
     </property>
    </widget>
    <widget class="QWidget" name="selectionColourPanel" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="selectionColourPickerButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>81</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Color Picker</string>
     </property>
    </widget>
    <widget class="QLabel" name="selectionColourLabel_r">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="selectionColourLabel_g">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>58</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="selectionColourLabel_b">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>58</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="selectionColourLabel_a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>58</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>a</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>rocketMetaCheckBox</tabstop>
  <tabstop>rowSpin</tabstop>
  <tabstop>columnSpin</tabstop>
  <tabstop>gridOrientationCombo</tabstop>
  <tabstop>horizontalInversionCheckBox</tabstop>
  <tabstop>verticalInversionCheckBox</tabstop>
  <tabstop>enableBoxesCheckBox</tabstop>
  <tabstop>systemWallpaperCheckBox</tabstop>
  <tabstop>wallpaperSelectorButton</tabstop>
  <tabstop>systemWallpaperScreenSpin</tabstop>
  <tabstop>systemWallpaperBlurSpin</tabstop>
  <tabstop>font1Spin</tabstop>
  <tabstop>font2Spin</tabstop>
  <tabstop>baseColourPickerButton</tabstop>
  <tabstop>secondaryColourPickerButton</tabstop>
  <tabstop>selectionColourPickerButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>systemWallpaperCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>wallpaperSelectorButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>systemWallpaperCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>systemWallpaperScreenSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>systemWallpaperCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>systemWallpaperScreenLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
